h/static/scripts/group-forms/components/GroupModeration.tsx

```diff
@@ -1,46 +1,80 @@
 import { Spinner } from '@hypothesis/frontend-shared';
 import classnames from 'classnames';
-import { useState } from 'preact/hooks';
+import { useEffect, useRef, useState } from 'preact/hooks';
 
 import type { Group } from '../config';
+import type { GroupAnnotationsResult } from '../hooks/use-group-annotations';
 import { useGroupAnnotations } from '../hooks/use-group-annotations';
-import type { APIAnnotationData } from '../utils/api';
 import { moderationStatusToLabel } from '../utils/moderation-status';
 import GroupFormHeader from './GroupFormHeader';
 import type { ModerationStatus } from './ModerationStatusSelect';
 import ModerationStatusSelect from './ModerationStatusSelect';
 import FormContainer from './forms/FormContainer';
 
+/**
+ * Checks if the Window's scroll is at the bottom.
+ *
+ * @param offset - Return true if the difference between the element's current
+ *                 and maximum scroll position is below this value.
+ *                 Defaults to 20.
+ */
+function windowScrollIsAtBottom(offset = 20): boolean {
+  const distanceToTop = window.scrollY + window.innerHeight;
+  const triggerPoint = document.body.clientHeight - offset;
+  return distanceToTop >= triggerPoint;
+}
```

Comment:
This works fine. Using only element APIs I believe this should work as well:

```js
const el = document.scrollingElement;
const scrollRemaining = el.scrollHeight - el.scrollTop - el.clientHeight
return scrollRemaining < threshold;
```

This does assume that you have `<!doctype html>` at the top of your HTML so you're not in quirks mode.